src/View/Helper/UrlHelper.php

```diff
@@ -98,8 +98,10 @@ public function buildFromPath(string $path, array $params = [], array $options =
     public function image(string $path, array $options = []): string
     {
         $options += ['theme' => $this->_View->getTheme()];
+        /** @var class-string $className */
+        $className = App::className('Asset', 'Routing');
```

Comment:
The asset classname should be a helper config similar to https://github.com/cakephp/cakephp/blob/41903eb1751b7c7c4dc45bbe9ceed7514cd52135/src/View/Helper/TextHelper.php#L49-L52

Reply:
Updated.